apps/full-site-editing/full-site-editing-plugin/donations/index.js

```diff
@@ -4,14 +4,102 @@
 import apiFetch from '@wordpress/api-fetch';
 import { getBlockType, registerBlockType, unregisterBlockType } from '@wordpress/blocks';
 import { __, _x } from '@wordpress/i18n';
-/* eslint-enable wpcalypso/import-docblock */
 
 /**
  * Internal dependencies
  */
 import edit from './edit';
+
+/**
+ * Style dependencies
+ */
 import './style.scss';
 
+const name = 'a8c/donations';
+
+const settings = {
+	title: __( 'Donations (a8c-only)', 'full-site-editing' ),
+	description: __(
+		'Collect one-time, monthly, or annually recurring donations.',
+		'full-site-editing'
+	),
+	attributes: {
+		currency: {
+			type: 'string',
+			default: 'USD',
+		},
+		showMonthly: {
+			type: 'boolean',
+			default: true,
+		},
+		showAnnual: {
+			type: 'boolean',
+			default: true,
+		},
+		showCustom: {
+			type: 'boolean',
+			default: true,
+		},
+		oneTimeHeading: {
+			type: 'string',
+			default: __( 'Make a one-time donation', 'full-site-editing' ),
+		},
+		monthlyHeading: {
+			type: 'string',
+			default: __( 'Make a monthly donation', 'full-site-editing' ),
+		},
+		annualHeading: {
+			type: 'string',
+			default: __( 'Make a yearly donation', 'full-site-editing' ),
+		},
+		chooseAmountText: {
+			type: 'string',
+			default: __( 'Choose an amount (USD)', 'full-site-editing' ),
+		},
+		customAmountText: {
+			type: 'string',
+			default: __( 'Or enter a custom amount', 'full-site-editing' ),
+		},
+		extraText: {
+			type: 'string',
+			default: __( 'Your contribution is appreciated.', 'full-site-editing' ),
+		},
+		oneTimeButtonText: {
+			type: 'string',
+			default: __( 'Donate', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **79** times:
`translate( 'Donate' )` *ES Score: 13*